crud/crud.cpp

```diff
@@ -398,21 +398,14 @@ crud::set_leader_info(bzn::message& msg)
 
 
 bool
-crud::validate_create(const bzn::message &msg)
+crud::validate_create_or_update(const bzn::message &msg)
```

Comment:
please move reference to the type throughout the code

Reply:
Done in crud.cpp, crud.h and crud_test.cpp